templates/webapi/NewJob/create.html.ep

```diff
@@ -0,0 +1,122 @@
+
+<div>
+  <h2><%= title %></h2>
+  
+  %= include 'layouts/info'
+  <p><b>Hint:</b> Experimental form to trigger a test E.g. see <%= link_to 'this
+								      link' => url_for('not_found') %>.
+  </p>
+  <div class="container-md">
+  <form class="row g-3" onsubmit="runExample(this);" data-post-url="/api/v1/isos">
+    <div class="col-md-6">
+      <label for="distri" class="form-label">
+	<strong>Distri</strong></label>
+      <input type="text" class="form-control" id="distri"
+	     placeholder="example" value="example" name="DISTRI">
+    </div>
+    <div class="col-md-6">
+      <label for="version" class="form-label">
+	<strong>Version</strong></label>
+      <input type="text" class="form-control" id="version" placeholder="0" value="0" name="VERSION">
+    </div>
+    <div class="col-6">
+      <label for="flavor" class="form-label">
+	<strong>Flavor</strong></label>
+      <input type="text" class="form-control" id="flavor" placeholder="DVD" value="DVD" name="FLAVOR">
+    </div>
+    <div class="col-6">
+      <label for="arch" class="form-label">
+	<strong>Arch</strong></label>
+      <select id="arch" class="form-select" name="ARCH">
+	<option></option>
+	<option selected>x86_64</option>
+	<option>arch64</option>
+      </select>
+    </div>
+    <div class="col-md-6">
+      <label for="test" class="form-label">
+	<strong>Test Name</strong></label>
+      <input type="text" class="form-control" id="test"
+	     placeholder="simple_boot" value="simple_boot" name="TEST">
+      </div>
+    <div class="col-md-6">
+      <label for="build" class="form-label">
+	<strong>Build</strong></label>
+      <input type="text" class="form-control" id="build" value="openqa" name="BUILD">
+    </div>
+    <div class="col-md-12">
+      <label for="casedir" class="form-label">
+	<strong>CASEDIR</strong></label>
+      <input type="text" class="form-control" id="casedir"
+	     placeholder="https://github.com/os-autoinst/os-autoinst-distri-example.git"
+	     value="https://github.com/os-autoinst/os-autoinst-distri-example.git" 
+	     name="casedir"  name="CASEDIR">
+    </div>
+    <div class="col-md-12">
+      <label for="needles_dir" class="form-label">
+	<strong>NEEDLES_DIR</strong></label>
+      <input type="text" class="form-control" id="needles_dir"
+	     placeholder="https://github.com/os-autoinst/os-autoinst-distri-example/tree/main/needles"
+	     name="NEEDLES_DIR">
+    </div>
+    
+    <div class="md-12">
+      <label for="scenario_definition" class="form-label col-sm-2
+					      control-label">
+	</strong>Scenario Definition</strong></label>
+      <textarea type="text" class="form-control" id="scenario_definition"
+		rows="15" name="SCENARIO_DEFINITIONS_YAML">
+---
+products:
+  example:
+    distri: "example"
+    flavor: "DVD"
+    arch: "x86_64"
+    version: '0'
+machines:
+  64bit:
+    backend: "qemu"
+job_templates:
+  simple_boot:
+    product: "example"
+    machine: "64bit"
+      </textarea>
+    </div>
+    
```

Comment:
I would leave this whole block out for now. openQA should read the scenario definitions from the example test distribution itself.

Reply:
But having this in the PoC makes it easy to test